python/src/aiconfig/default_parsers/openai.py

```diff
@@ -50,6 +56,8 @@ async def serialize(
         Returns:
             str: Serialized representation of the prompt and inference settings.
         """
+        event = CallbackEvent("on_serialize_start", {"input": locals()})
```

Comment:
locals?

Reply:
`locals()`  gets the function params. Updating this to the relevant input params, discussed offline w/ @saqadri

Reply:
Curious for @saqadri 's thinking. My initial reaction was "there's a ton of stuff in there we may not want or need to log."